Objects/object.c

```diff
@@ -2134,6 +2134,13 @@ _PyTrash_end(PyThreadState *tstate)
 }
 
 
+int
```

Comment:
Please add comment explaining that it is used by the Py_TRASHCAN_BEGIN macro.

Reply:
Got it, updated.